.github/workflows/stage-build.yml

```diff
@@ -235,14 +238,18 @@ jobs:
           BLOG_ROOT: ${{ github.workspace }}/mdn/mdn-studio/content/posts
           CURRICULUM_ROOT: ${{ github.workspace }}/mdn/curriculum
           GENERIC_CONTENT_ROOT: ${{ github.workspace }}/mdn/generic-content/files
+          YARI_ROOT: ${{ github.workspace }}/mdn/yari
           BASE_URL: "https://developer.allizom.org"
 
           # rari
-          BUILD_OUT_ROOT: "client/build"
+          BUILD_OUT_ROOT: "out"
           LIVE_SAMPLES_BASE_URL: https://live.mdnyalp.dev
           INTERACTIVE_EXAMPLES_BASE_URL: https://interactive-examples.mdn.allizom.net
           ADDITIONAL_LOCALES_FOR_GENERICS_AND_SPAS: de
 
+          FRED_ROBOTS_GLOBAL_ALLOW: false
```

Comment:
Nit: I would hope for this to be the default, to avoid accidental indexing of stage/test/3rd party environments.